src/aca_main.cpp

```diff
@@ -72,6 +72,15 @@ std::atomic_ulong g_total_update_GS_time(0);
 
 bool g_demo_mode = false;
 bool g_debug_mode = false;
+int processor_count = std::thread::hardware_concurrency();
+/*
+  From previous tests, we found that, for x number of cores,
+  it is more efficient to set the size of both thread pools
+  to be x * (2/3), which means the total size of the thread pools
+  is x * (4/3). For example, for a host with 24 cores, we would 
```

Comment:
Maybe we need to fix comment from 4/3 to 2/3

Reply:
Yes and no. The size of each thread pool is `x * (2/3)`, and there are two thread pools, one for the on-demand engine, the other for the async gRPC server. So the total size of the thread pools is ` 2 * (x * (2/3)) = x * (4/3)`.